cmd/output/notify.go

```diff
@@ -4,7 +4,7 @@ import "fmt"
 
 var notifyCmdMovedBody = AnsiFaint + "A new set of commands related to apps in organizations have been promoted as \n" +
 	"the default commands, and previous commands have been moved to the \"legacy\" namespace." + AnsiReset + "\n\n" +
-	"See https://www.numerous.com/docs/legacy-cli-commands for more information."
+	"See http://www.numerous.com/docs/cli#legacy-commands for more information."
```

Comment:
isn't it `https`? Think you may changed it by accident

Reply:
fixed :)